HW02_B_Mimic_starter.Rmd

```diff
@@ -91,7 +113,15 @@ The above graph lets you see some colobrlind friendly palettes. For the graph be
 Now using the above mpg dataset, make this graph
 
 ```{r graph 4 code}
-
+  ggplot(mpg, aes(cty, class))+
+  geom_jitter(aes(color=class))+
+  geom_boxplot(alpha=0.2)+
+  scale_color_brewer(palette="Set2") +
+  labs(title="Horizontal BoxPlot of City MPG and Car Class", x = "Car Class", y = "City mpg") +
+  theme(panel.grid.major = element_blank(), panel.grid.minor=element_blank(), panel.background=element_blank(), panel.border=element_blank(), axis.line = element_line(color="black"))
+
+#I think my plot has more data points than the image...not sure why

```

Comment:
At first, my graph had less points than the graph, so I used jitter to try to add variance. Doing that gave me too many points as well. I remember that you can change the degree to which your points are jittered, maybe that is the solution here? Not completely sure though.